### PR TITLE
Modify README.md files of tutorials

### PR DESCRIPTION
**Pull request by Nipunich**
## Purpose
Modify Logging using Logstash and Kibana README.md file and querying-mysql-database README.md file.

### CODE REVIEW

File: docs/content/src/observability/logstash-kibana.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Logging using Logstash and Kibana
 
+Ballerina distributed logging and analysis is supported by Elastic Stack.
```
Comment (samgnaniah) [inline docs/content/src/observability/logstash-kibana.md]: Let's add the link to Elastic Stack here and remove it from line 12.

File: docs/content/src/observability/logstash-kibana.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+The elastic stack comprises of the following components.
+
+- Beats - Multiple agents that ship data to Logstash or Elasticsearch. In our context, Filebeat will ship the Ballerina logs to Logstash. Filebeat should be a container running on the same host as the Ballerina service. This is so that the log file (ballerina.log) can be mounted to the Filebeat container.
+- Logstash - Used to process and structure the log files received from Filebeat and send to Elasticsearch.
+- Elasticsearch - Storage and indexing of the logs received by Logstash.
+- Kibana - Visualizes the data stored in Elasticsearch
+
 Ballerina has a log module for logging to the console. You can import the `ballerina/log` module and start logging. The following section will describe how to search, analyze, and visualize logs in real time using [Elastic Stack](https://www.elastic.co/).
 
 To understand how you can track logging for Ballerina services, let’s consider a service that converts JSON to XML.
```
Comment (samgnaniah) [inline docs/content/src/observability/logstash-kibana.md]:
```suggestion
To understand how you can track logging for Ballerina services, let’s consider the service used in the quick start guide.
```

Let's fix this as well.

Comment (samgnaniah) [inline docs/content/src/observability/logstash-kibana.md]:
```suggestion
To understand how you can track logging for Ballerina services, let’s consider the service used in the quick start guide.
```

Let's update this as well. Please update step 3 of Set up the project section as well. This is not XML to JSON transformation so we need to change this to say we are using the service in the quick start.

File: docs/content/src/observability/logstash-kibana.md (hunk 1 above)
Comment (Nipunich) [reply docs/content/src/observability/logstash-kibana.md]: Fixed

File: docs/content/src/observability/logstash-kibana.md [hunk 3]
```diff
@@ -1,5 +1,14 @@
 # Logging using Logstash and Kibana
 
+Ballerina distributed logging and analysis is supported by Elastic Stack.
+
+The elastic stack comprises of the following components.
```
[the remark this answers is not in the document]
Comment (Nipunich) [reply docs/content/src/observability/logstash-kibana.md]: Fixed

File: docs/content/src/observability/logstash-kibana.md (hunk 2 above)
Comment (Nipunich) [reply docs/content/src/observability/logstash-kibana.md]: Fixed

**Pull request merged**